### PR TITLE
make graph as default cmmand and connections as alias

### LINKED ISSUE #6473: rename `rad app connections` to `rad app graph`
## Overview of feature request

We've gotten feedback that `rad app connections` is a confusing name. What we're really showing is the application graph, not just the connections. Since we're using the term application graph more often to talk about Radius this also makes a good command name.

```
rad app graph
```

Since this is coming in late for public release we should add an alias for now and rename the command in v0.27. 

## Acceptance criteria

**Public Release**

- Add alias to `rad app graph`. This has no risk and `rad app connections` will still work.

**v0.27**

- Rename the command to `rad app graph`. Add an alias back to `rad app connections` so that it will still work.
- Update documentation and tutorial

## Additional context

We want to add this alias in time for the demo.

[AB#9727](https://dev.azure.com/azure-octo/e61041b4-555f-47ae-95b2-4f8ab480ea57/_workitems/edit/9727)

### PR DESCRIPTION
**Pull request by nithyatsu**
# Description

Change "default" command to rad app graph, so that the auto-generated docs use that instead of rad app connections.

## Type of change

- This pull request fixes a bug in Radius and has an approved issue (issue link required).

Fixes: #6473

### CODE REVIEW

File: pkg/cli/cmd/app/graph/graph.go [hunk 1]
```diff
@@ -24,17 +24,17 @@ import (
 func NewCommand(factory framework.Factory) (*cobra.Command, framework.Runner) {
 	runner := NewRunner(factory)
 	cmd := &cobra.Command{
-		Use:     "connections",
-		Aliases: []string{"graph"},
-		Short:   "Shows the connections for an application.",
-		Long:    `Shows the connections for an application`,
+		Use:     "graph",
+		Aliases: []string{"connections"},
```
Comment (nithyatsu) [inline pkg/cli/cmd/app/graph/graph.go:28]: do we want connections as alias ?

Comment (vinayada1) [reply pkg/cli/cmd/app/graph/graph.go:28]: Does not make sense to me but better to check with PM.

File: pkg/cli/cmd/app/graph/graph_test.go [hunk 2]
```diff
@@ -83,7 +83,7 @@ func Test_Validate(t *testing.T) {
 			},
 		},
 		{
-			Name:          "Connections command missing application",
+			Name:          "Graphs command missing application",
```
Comment (vinayada1) [inline pkg/cli/cmd/app/graph/graph_test.go:86]: nit: Graph instead of Graphs

File: pkg/cli/cmd/app/graph/graph.go [hunk 3]
```diff
@@ -24,17 +24,17 @@ import (
...
+		Short:   "Shows the application graph for an application.",
+		Long:    `Shows the appliaction graph for an application`,
 		Args:    cobra.MaximumNArgs(1),
 		Example: `
 # Show connections for current application
-rad app connections
+rad app graph
 
 # Show connections for specified application
```
Comment (vinayada1) [inline pkg/cli/cmd/app/graph/graph.go:36]: Fix comment

File: cmd/rad/cmd/root.go [hunk 4]
```diff
@@ -283,7 +283,7 @@ func initSubCommands() {
 	appStatusCmd, _ := app_status.NewCommand(framework)
 	applicationCmd.AddCommand(appStatusCmd)
 
-	appConnectionsCmd, _ := app_connections.NewCommand(framework)
+	appConnectionsCmd, _ := app_graph.NewCommand(framework)
```
Comment (vinayada1) [inline cmd/rad/cmd/root.go:286]: nit: rename appConnectionsCmd

File: pkg/cli/cmd/app/graph/graph.go [hunk 5]
```diff
@@ -24,17 +24,17 @@ import (
...
-		Aliases: []string{"graph"},
-		Short:   "Shows the connections for an application.",
-		Long:    `Shows the connections for an application`,
+		Use:     "graph",
+		Aliases: []string{"connections"},
+		Short:   "Shows the application graph for an application.",
+		Long:    `Shows the appliaction graph for an application`,
 		Args:    cobra.MaximumNArgs(1),
 		Example: `
 # Show connections for current application
```
Comment (vinayada1) [inline pkg/cli/cmd/app/graph/graph.go:33]: Fix comment

File: pkg/cli/cmd/app/graph/graph.go (hunk 1 above)
Comment (AaronCrawfis) [reply pkg/cli/cmd/app/graph/graph.go:28]: Yes let's keep it to preserve backwards compatibility

**Review by vinayada1 — APPROVED** [review]

**Review by rynowak — APPROVED** [review]
:shipit:

**Comment by nithyatsu** [comment]
@rynowak @vinayada1 could you please approve functional test. I had to just rebase it on latest main.

File: pkg/cli/cmd/app/graph/graph.go (hunk 1 above)
Comment (rynowak) [reply pkg/cli/cmd/app/graph/graph.go:28]: @AaronCrawfis wants to keep it around. Let's log an issue, we can decide later when to address it.

Comment (nithyatsu) [reply pkg/cli/cmd/app/graph/graph.go:28]: https://github.com/radius-project/radius/issues/7262

**Review by youngbupark — APPROVED** [review]

**Pull request merged**